### PR TITLE
update render readme yml

### PR DESCRIPTION
**Pull request by jdhoffa**

### CODE REVIEW

File: .github/workflows/render-readme.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: macOS-latest
     steps:
       - uses: actions/checkout@v2
```
Comment (cjyetman) [inline .github/workflows/render-readme.yaml:18]:
```suggestion
      - uses: actions/checkout@v3
```

File: .github/workflows/render-readme.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: macOS-latest
```
Comment (cjyetman) [inline .github/workflows/render-readme.yaml:16]: might be better (cheaper) to run this on ubuntu

File: .github/workflows/render-readme.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
-      - uses: r-lib/actions/setup-r@v1
+      - uses: r-lib/actions/setup-r@v2
       - uses: r-lib/actions/setup-pandoc@v1
```
Comment (cjyetman) [inline .github/workflows/render-readme.yaml:20]:
```suggestion
      - uses: r-lib/actions/setup-pandoc@v2
```

File: .github/workflows/render-readme.yaml (hunk 2 above)
Comment (jdhoffa) [reply .github/workflows/render-readme.yaml:16]: I believe it's all free since the package is public

**Comment by jdhoffa** [comment]
Leaving as a draft for now, as I try to figure out why the yml isn't working.

**Comment by jdhoffa** [comment]
Closing since stale

**Pull request closed without merging**